iphone/cli/commands/_build.js

```diff
@@ -4638,7 +4638,7 @@ iOSBuilder.prototype.copyTitaniumiOSFiles = function copyTitaniumiOSFiles() {
 						// The content of Frameworks directory only change if we change SDK version. So it is safe to copy whole directory.
 						// Copy whole 'Frameworks' directory from SDK to build directory, to preserve symlink available in Titaniumkit.xcframework.
 						// TODO: Is there any better way to do this?
-
+						fs.emptyDirSync(path.join(this.buildDir, dir));
```

Comment:
may want to stick a comment in the code why we're doing this so it doesn't get reverted way down the road.

Reply:
Had missed this comment, just pushed up a commit now that explains why we nuke before copying